circle.yml

```diff
@@ -0,0 +1,21 @@
+machine:
+  xcode:
+    version: "8.0"
+
+dependencies:
+  pre:
+    - brew update 1> /dev/null 2> /dev/null
+    - brew outdated carthage || brew upgrade carthage
+
+checkout:
+  post:
+    - git submodule update --init --recursive
+test:
+  override:
+    - ./scripts/macOS
```

Comment:
Yeah I was going to do that but then I got lazy but I think I'm going to again today. Might just end up using fast lane because it's already installed on the machine